proseco/core.py

```diff
@@ -691,6 +691,8 @@ def plot(self, ax=None, **kwargs):
         :param kwargs: other keyword args for plot_stars
         """
         kwargs.setdefault('catalog', self.get_catalog_for_plot())
+        if self.stars is None:
```

Comment:
It looks like set_stars has its own logic to try to decide where to *get* the stars, but I think it is relatively low-impact to just have a go at getting something if we reach this point and have no stars.

Unless this should be done from BaseCatalogTable or from chandra_aca.plot itself...